### DISCUSSION TITLE
How to install Slack plugin?

### DISCUSSION THREAD

**Discussion by geoidesic** [fluent/fluent-bit#8911] [Q&A]
Hi, I'm using Alma Linux 9. There's a dnf distribution of fluent-bit, which I installed. However when I tried to confiture it for slack, it didn't recognise that as a plugin. So questions:

1. How can I see what plugins are enabled?
2. How can I install or enable the slack plugin? 

I tried building from source but (typically) `cmake` failed due to missing, uninstallable dependencies.

**Comment by patrick-stephens** [comment]
Please use the official packages provided via the Fluent Bit repositories: https://docs.fluentbit.io/manual/installation/linux/redhat-centos
These should work for AlmaLinux too.

We cannot support any custom builds by package maintainers directly really.
If you run with `--help` it will give you full details of configuration and plugins supported.
It sounds like it was built without support for Slack or from an earlier version.
